crates/types/src/lib.rs

```diff
@@ -1,3 +1,7 @@
 pub mod operator;
+pub mod scope;
+pub mod sessionid;
```

Comment:
Should probably make these not-pub since we're reexporting the one type we want to use from them.